lib/SILOptimizer/UtilityPasses/NonInlinableFunctionSkippingChecker.cpp

```diff
@@ -65,6 +65,12 @@ static bool shouldHaveSkippedFunction(const SILFunction &F) {
   if (isa<DestructorDecl>(func) || isa<ConstructorDecl>(func))
     return false;
 
+  // See DeclChecker::shouldSkipBodyTypechecking. Can't skip didSet for now.
```

Comment:
Should I try change SILGen to skip generating SIL in this one case? It still won't output it, but will currently generate it. Didn't seem worth adding another check in there though.

Reply:
Yeah, I think it would be better for SILGen to make its own determination about what should be emitted instead of seeing if something has been type checked or not.

Reply:
I guess we could add a check into SILGenModule::emitFunction, but I'm not sure what it would check for other than this one case. Are there any other cases where we would want to type check something but not generate SIL for it?

May also be worth just putting this in as a workaround to rdar://70739449 for now and opening another radar to change SILGen later? Up to you all :) (@CodaFi @slavapestov)